mobify/platform-scaffold
# Unify address fields in Billing and Shipping

# Unify address fields in Billing and Shipping
It appears that the label and order of the address fields for Billing and Shipping are not identical. This tiny PR fixes the issue.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-89

## Changes
- Change the label and order of address field in Billing and Shipping


## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=0)
- Add a product to cart
- Go to Checkout
- Verify the label and order of shipping address fields
- Click "continue to payment"
- Uncheck "same as shipping address"
- Verify the label and order of billing address fields

Expected results:
Both shipping and billing address pages should follow this order and these lablels:
Full Name
Address
City
Country
State/Provice
Zip/Postal Code
